runtime/common/src/apis.rs

```diff
@@ -263,17 +263,32 @@ macro_rules! impl_runtime_apis_plus_common {
 					let is_transactional = false;
 					let validate = true;
 
-					let gas_limit = if gas_limit > U256::from(u64::MAX) { u64::MAX } else { gas_limit.low_u64() };
+					let estimated_transaction_len = (from.encode().len() +
```

Comment:
Please add all constant size mandatory fields in the constant, like from to value, gas limit, etc